### PR TITLE
Posts - More link: replace angular quotation mark with arrow

### PR DESCRIPTION
**Pull request by eliorivero**
The angular quotation mark is a punctuation character in spanish so it looks odd to use it as a UI element. Despite the fact that the theme is in english originally, it's necessary to update this since this character is in the CSS and can't be translated.

This commit replaces it with an arrow, more appropriate for any language. Additionally, this PR also removes a duplicated `display` property.

### Before

captura de pantalla 2018-10-27 a la s 21 34 19

### After

captura de pantalla 2018-10-27 a la s 21 33 58

### Testing instructions

1. Checkout this branch and build with `npm install && npm run build:style`
2. add a new post
3. add a paragraph, then the More separator, and then another paragraph
    captura de pantalla 2018-10-27 a la s 21 45 23
4. publish and check in post list view

### CODE REVIEW

File: sass/site/primary/_posts-and-pages.scss [hunk 1]
```diff
@@ -168,7 +168,6 @@
 				height: 100%;
 				top: 0; left: 0;
 				content: "\020";
-				display: block;
```
Comment (kjellr) [inline sass/site/primary/_posts-and-pages.scss]: Is this left over from a prior commit? Otherwise, this looks good!

Comment (eliorivero) [reply sass/site/primary/_posts-and-pages.scss]: Yeah it's unrelated, I just happened to notice it and removed it. There's already a display: block in line 166.

captura de pantalla 2018-10-29 a la s 12 54 32

Comment (kjellr) [reply sass/site/primary/_posts-and-pages.scss]: Confirmed. 👍 Thanks for the fix.

**Comment by eliorivero** [comment]
Thanks for the review and merge!

**Pull request merged**